jetty-io/src/main/java/org/eclipse/jetty/io/ssl/SslConnection.java

```diff
@@ -88,7 +88,7 @@
     private ByteBuffer _decryptedInput;
     private ByteBuffer _encryptedInput;
     private ByteBuffer _encryptedOutput;
-    private final boolean _encryptedDirectBuffers = false;
+    private final boolean _encryptedDirectBuffers = true;
```

Comment:
I would prefer if this default was not changed.  IE we can merge the code, but you will have to configure this to true yourself

Reply:
The problem is none of the other changes in this PR matter for fixing it in Android 8.1 - they just happened to be in cherry picked commit. Can you suggest an alternative approach of exposing this to override it? We can ship our own custom build of jetty, but I was just trying to avoid that. Thanks for your help with this!

Reply:
@wakkow, I think if your PR removes the interceptor stuff, you will be pretty close.

Reply:
@gregw Perhaps this could be a ServerConnector configuration that is passed into the Connections? (with a default of false in the ServerConnector)